java/src/test/java/ai/rapids/cudf/ColumnVectorTest.java

```diff
@@ -527,6 +527,46 @@ void testSpark32BitMurmur3HashDoubles() {
     }
   }
 
+  @Test
+  void testSpark32BitMurmur3HashTimestamps() {
+    try (ColumnVector v = ColumnVector.timestampMicroSecondsFromBoxedLongs(
+        0L, null, 100L, -100L, 0x123456789abcdefL, null, -0x123456789abcdefL);
+         ColumnVector result = ColumnVector.spark32BitMurmurHash3(42, new ColumnVector[]{v});
+         ColumnVector expected = ColumnVector.fromBoxedInts(-1670924195, 42, 1114849490, 904948192, 657182333, 42, -57193045)) {
```

Comment:
can we have a comment pointing to https://github.com/rapidsai/cudf/pull/7672/files#diff-4eb211c99e029b2b29bc31fb5e890a8c1be89b4d9cbe60f3d4dbc9b33a0042abR281 explaining how expected values were calculated

Reply:
I missed this before it merged, but I will be posting a new PR shortly with struct support and can add the comment there.